dev/docker/trino/Dockerfile

```diff
@@ -7,44 +7,6 @@ LABEL maintainer="support@datastrato.com"
 
 USER root
 
-# Only mysql, hudi, iceberg, mariadb, jmx, memory, tpch, tpcds, postgresql, hive plugin are kept
-RUN rm -rf /usr/lib/trino/plugin/accumulo  \
```

Comment:
Why are you removing these lines? Can you leave a brief comment about the reason behind it?

Reply:
Okay, because Docker image use Overlay2 FS, which will treat any file operations as a new layer, for delete operation, those files are marked as deleted, but the actual file blob is not removed, that's why if we deleted those files, the size of docker image won't shrink.

Reply:
hi @unknowntpo Thank you force this issue.
Is there any way to remove these unused file and reduce the size of Docker image?

Reply:
I'm wrong, I thought these plugins takes not much space, but actually, it takes almost 800 MB

<img width="434" alt="image" src="https://github.com/datastrato/gravitino/assets/10594519/13af21d8-80b7-453f-b7b7-9a645b603f3e">

Reply:
So, it worth it to use multi-stage build to build a slim Trino image, just like what we did in `hive`, `doris`, let me work on it.

Reply:
I think removing the plugin won't save much space, at most about 300MB. Moreover, managing the images is not conducive to multi-version testing of Trino.

Reply:
The original size of `datastrato/gravitino-ci-trino` is  1.32GB, so  I think reduce 300MB is a lot.

> Moreover, managing the images is not conducive to multi-version testing of Trino.

But in current implementation, we only run IT tests with `trinodb/trino:426`, why do we need multi-version testing ?

>  This docker image is only use in the play ground

And actually, this image is for CI tests, not for `gravitino-playground`.